style.css

```diff
@@ -113,3 +136,222 @@ body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  color: #344563;
+  margin-right: auto;
+  margin-left: 16px;
+  line-height: 16px;
+}
+
+.back-end {
+  position: relative;
+  right: 0.6rem;
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.year {
+  position: relative;
+  right: -1.3rem;
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.small-text-2 {
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  height: 295px;
+  width: 295px;
+}
+
+.language {
+  display: flex;
+  align-items: flex-start;
+  list-style-type: none;
+  color: #6070ff;
+  position: relative;
+  left: -2.5rem;
+  margin-top: 12px;
+}
+
+.language-type {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.project-button {
+  color: #396df2;
+  background-color: transparent;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  cursor: pointer;
+  padding: 12px;
+}
+
+.about-bigger-text {
+  width: 327px;
+  height: 35px;
+  color: #172b4d;
+  font-size: 40px;
+  font-weight: 700;
+  font-family: "Poppins", sans-serif;
+  line-height: 52px;
+  margin: 114px 24px auto 24px;
+}
+
+.about-text {
+  width: 327px;
+  height: 147px;
+  font-size: 16px;
+  font-weight: 400;
+  font-family: "Poppins", sans-serif;
+  list-style: 24px;
+  color: #344563;
+  margin: 12px 24px auto 24px;
+}
+
+.about-list {
+  display: flexbox;
+  list-style: none;
+  padding: 10px;
+  margin-left: 24px;
+  margin-right: 36px;
+}
+
+.resume-button {
+  color: #396df2;
+  background-color: transparent;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  cursor: pointer;
+  padding: 12px;
+  margin-left: 24px;
+  margin-bottom: 48px;
+}
+
+.language-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.language-text {
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  margin-bottom: 36px;
+}
+
+.drop-down {
+  display: flex;
+  margin-left: auto;
+  text-align: center;
+  background-color: transparent;
+  font-size: large;
+  margin-top: -1rem;
+  margin-bottom: 1em;
+  border: none;
+}
+
+.frame-lang {
+  list-style-type: none;
+  background-color: #f7f7f9;
+  margin-bottom: 12px;
+  padding: 9px 0 10px 12px;
+  display: flex;
+  align-items: center;
+  border-radius: 8px;
+}
+
+.tab-text {
+  margin-left: 16px;
+}
+
+.unopened-title {
+  padding: 24px 0;
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.language-words {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+button:hover {
```

Comment:
- [ ] I kindly suggest that you add the correct button interactions for hover as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on hover has the `background-color: #6070FF;` not `background-color: #396df2;`

### EXAMPLE:
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```